yoyodyne/collators.py

```diff
@@ -28,8 +27,8 @@ def __init__(
         pad_idx,
         config: dataconfig.DataConfig,
         arch: str,
-        max_source_length: int = 128,
-        max_target_length: int = 128,
+        max_source_length=defaults.MAX_SOURCE_LENGTH,
```

Comment:
Is removing the type annotation here intentional?

Reply:
Reverted.